starter-code/src/viking.js

```diff
@@ -1,11 +1,108 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damageAmmount) {
+    this.health = this.health - damageAmmount;
```

Comment:
ici tu peux utiliser -=    =>   this.health -= damageAmmount; 
Mais ce n'est en aucun cas obligatoire, ton code fonctionne très bien comme ca.